portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,35 +34,42 @@
 public class DataServlet extends HttpServlet {
 
   private ArrayList<String> messages = new ArrayList<String>();
+  private ArrayList<String> firstName = new ArrayList<String>();
+  private ArrayList<String> lastName = new ArrayList<String>();
```

Comment:
These three are related. It would be good to combine them into a data class, say Comment.

Imagine if later you have to add one or more fields (say timestamp), if you start with a class you can easily add fields. Otherwise, you will have to start adding ArrayList for each of them.

Reply:
I forgot to take that out, I was showing Ade something yesterday and just for got to take it out. I don't need any of those lists since I am using datastore, it is just dead code now. Thank you for pointing those out, I will take it out.